### ISSUE TITLE
feat(a11y): Buttons not reachable with keyboard

### ISSUE THREAD

**Issue by cguhl-adesso** [Rudo-hayate/open-webui-a11y#29]
### Check Existing Issues

- [x] I have searched for all existing **open AND closed** issues and discussions for similar requests. I have found none that is comparable to my request.

### Verify Feature Scope

- [x] I have read through and understood the scope definition for feature requests in the Issues section. I believe my feature request meets the definition and belongs in the Issues section instead of the Discussions.

### Problem Description

- in chat-navigation there is a button (...) which is not reachable with keyboard

<img width="244" height="135" alt="Image" src="https://github.com/user-attachments/assets/66e009c8-ce27-4aea-9dea-043c1d734841" />

### Desired Solution you'd like

- add navigation possibility for keyboard users

**Comment by cguhl-adesso** [comment]
Already fixed with https://github.com/Rudo-hayate/open-webui-a11y/issues/10
